### ISSUE TITLE
Copyediting

### ISSUE THREAD

**Issue by cooljeanius** [cooljeanius/The_Earths_Gut#7]
Some general minor copyediting issues I noticed throughout:
- [x] Miscapitalization
- [x] Spaces missing before or after ellipsises
- [ ] Wording

**Comment by cooljeanius** [comment]
I think this is mostly done, but I'll leave it open for another release

**Comment by cooljeanius** [comment]
Note that there're some instances of trolls talking oddly that I've noticed, but left in anyways because they're trolls

**Comment by cooljeanius** [comment]
Now that I've managed to generate an initial potfile for this campaign, progress can be judged by how many of the translatable strings in it look correct.

**Comment by cooljeanius** [comment]
ok I have completed an initial pass at making sure all translatable strings in the potfile look correct; I had to leave a few TODOs in places where I couldn't think of how to improve the wording, though... Leaving this open for some combination of the following happening:
- [ ] me completing another playthru
- [ ] me making a new release
- [ ] a translator to a new language being able to understand the text well enough to complete a new translation

**Comment by cooljeanius** [comment]
> I had to leave a few FIXMEs in places where I couldn't think of how to improve the wording, though...

To be specific:
- [ ] S08:
```
# po: FIXME: this sentence is worded kind of awkwardly and I'm not quite sure how to parse it myself; any clarifications are welcome:
story= _ "Hamel and his friends were lucky. The rest on their way to the main hall none wanted to fight them again. They took a break and made themselves familiar with the new surroundings."
```
- [ ] also S08:
```
# po: FIXME: phrasing is awkward here; suggestions for improvements are welcome:
message=_"Anyway, it seems they left me here back or rather have been forced to do so."
```
- [ ] S11t:
```
# po: FIXME: awkward wording; suggestions for improvements are welcome:
story= _ "Overwhelmed by the impression to regain their lost friend, the group returned to the main hall. They told each other everything that had transpired during a real long dwarvish banquet. Both parties were amazed about how far they had progressed in making new alliances."
```

**Comment by cooljeanius** [comment]
> ok I have completed an initial pass at making sure all translatable strings in the potfile look correct; I had to leave a few FIXMEs in places where I couldn't think of how to improve the wording, though... Leaving this open for some combination of the following happening:
> 
>     * [x]  me completing another playthru
> 
>     * [x]  me making a new release
> 
>     * [ ]  a translator to a new language being able to understand the text well enough to complete a new translation

So, I've done 2 of these now, but the things in the follow-up comment still need to be done, though... postponing any further work on this until 1.16.

**Comment by cooljeanius** [comment]
un-postponing

**Comment by cooljeanius** [comment]
some of the translation work that @k-konami has done for #22 has helped here, too

**Comment by cooljeanius** [comment]
@TheMythcaller might be helping with this one

**Comment by cooljeanius** [comment]
Postponed until @TheMythcaller returns

**Comment by cooljeanius** [comment]
un-postponed, as @Konrad22 has been helping out with this a bit on Discord

**Comment by cooljeanius** [comment]
> > I had to leave a few FIXMEs in places where I couldn't think of how to improve the wording, though...
> 
> To be specific:
> 
>    * [x]  S08:
> 
> 
> ```
> # po: FIXME: this sentence is worded kind of awkwardly and I'm not quite sure how to parse it myself; any clarifications are welcome:
> story= _ "Hamel and his friends were lucky. The rest on their way to the main hall none wanted to fight them again. They took a break and made themselves familiar with the new surroundings."
> ```
> 
>    * [x]  also S08:
> 
> 
> ```
> # po: FIXME: phrasing is awkward here; suggestions for improvements are welcome:
> message=_"Anyway, it seems they left me here back or rather have been forced to do so."
> ```
> 
>    * [ ]  S11t:
> 
> 
> ```
> # po: FIXME: awkward wording; suggestions for improvements are welcome:
> story= _ "Overwhelmed by the impression to regain their lost friend, the group returned to the main hall. They told each other everything that had transpired during a real long dwarvish banquet. Both parties were amazed about how far they had progressed in making new alliances."
> ```

OK I've checked off the first two of these now

**Comment by cooljeanius** [comment]
Depends on #45; will close this once @Konrad22 has completed his playthru and all text-related issues he opens are closed

**Comment by cooljeanius** [comment]
> Depends on #45; will close this once @Konrad22 has completed his playthru and all text-related issues he opens are closed

ok so that's closed, so I'm going to close this now.
